Makefile

```diff
@@ -14,9 +14,9 @@ GO ?= go
 ARGS ?= --strict_message --strict_error
 GOLANGCI_LINT_VERSION ?= v1.64.8
 # Set to use a different version of protovalidate-conformance.
-# Should be kept in sync with the version referenced in proto/buf.lock and
+# Should be kept in sync with the version referenced in buf.yaml and
 # 'buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go' in go.mod.
```

Comment:
I am wondering if we can/should advertise on the main readme the conformance version this library is targeting

Reply:
I like that idea

Reply:
Would definitely be helpful. Perhaps even more so to have it in the release notes.